elkc7sn/ansible/files/elasticsearch-template.conf

```diff
@@ -0,0 +1,169 @@
+{
```

Comment:
This file is the default template configuration that `logstash` provide for initializing new ES indexes. I have amended it to show how to force certain fields to not be analyzed.